File: Corte1/Tarea 120826Entrega140826/sensormentiroso.py
lecturas = [20, -999, 22, 24, -999, 26]
contadorMentiras = 0
contadorDatos = 0
sumaDatos = 0
for i in lecturas:
    if i == -999:
       contadorMentiras += 1
    else:
        contadorDatos += 1
        sumaDatos += i

print(f"Total de datos falsos: {contadorMentiras}")
print(f"Total de datos válidos: {contadorDatos}")
if contadorDatos > 0:
    print(f"Promedio de datos válidos: {sumaDatos / contadorDatos}")

File: Corte1/Tarea 120826Entrega140826/sensormentiroso.cpp
#include <iostream>
using namespace std;

int main(){
    int lecturas[6] = {20, -999, 22, 24, -999, 26};
    int cantidadDatos = 6;
    int contadorDatos = 1;
    int sumaDatos = lecturas[0];
    int contadorMentiras = 0;

    for (int i=1; i<cantidadDatos; i++){
        if (lecturas[i] != -999){
            sumaDatos += lecturas[i];
            contadorDatos++;
        } else {
            contadorMentiras++;
        }
    }
    cout << "El promedio de las lecturas es: " << sumaDatos/contadorDatos << endl;
    cout << "Los datos falsos son: " << contadorMentiras << endl;
    return 0;
}